src/components/DatePicker/index.android.js

```diff
@@ -18,6 +18,9 @@ class DatePicker extends React.Component {
 
         this.showPicker = this.showPicker.bind(this);
         this.setDate = this.setDate.bind(this);
+
+        this.maxDate = this.props.maxDate || new Date(CONST.DATE.MAX_DATE);
```

Comment:
Too many repetition, lets move new `new Date(CONST.DATE.MAX_DATE)` and  `new Date(CONST.DATE.MIN_DATE)` to default props